src/main/java/com/axonivy/market/factory/ProductFactory.java

```diff
@@ -68,6 +73,21 @@ public static Product mappingByMetaJSONFile(Product product, GHContent ghContent
 		return product;
 	}
 
+  private static MultilingualismValue mappingMultilingualismValueByMetaJSONFile(List<DisplayValue> list) {
+    MultilingualismValue value = new MultilingualismValue();
+    if (!CollectionUtils.isEmpty(list)) {
+      for (DisplayValue name : list) {
+        if (Language.EN.getValue().equalsIgnoreCase(name.getLocale())) {
+          value.setEn(name.getValue());
+        } else if (Language.DE.getValue().equalsIgnoreCase(name.getLocale())) {
+          value.setDe(name.getValue());
+        }
```

Comment:
Handle for not matching any language case

Reply:
We will check the structure of the meta JSON file on the repo market with the Jenkins pipeline, so I think it's not necessary to handle this case.